src/libs/costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -108,133 +119,146 @@ Costmap2DROS::Costmap2DROS(const std::string& name, tf2_ros::Buffer& tf) :
 
   // check if we want a rolling window version of the costmap
   bool rolling_window, track_unknown_space, always_send_full_costmap;
-  private_nh.param("rolling_window", rolling_window, false);
-  private_nh.param("track_unknown_space", track_unknown_space, false);
-  private_nh.param("always_send_full_costmap", always_send_full_costmap, false);
+
+  rolling_window = private_parameters_client->get_parameter<bool>("rolling_window", false);
```

Comment:
When you get rid of the parameters client, these would become
```c++
nh->get_parameter_or("rolling_window", rolling_window, false);
...
```

Reply:
#119